crmprtd/wmb_dir/normalize.py

```diff
@@ -0,0 +1,77 @@
+# Standard module
+import sys
+import os
+import logging
+import csv
+
+from datetime import datetime
+
+# Local
+from crmprtd.wmb import ObsProcessor, DataLogger
+
+# Installed libraries
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+def save_file(reader, cache_dir, data):
+    # save the downloaded file
+    fname_out = os.path.join(cache_dir, 'wmb_download' + datetime.strftime(datetime.now(), '%Y-%m-%dT%H-%M-%S') + '.csv')
+    with open(fname_out, 'w') as f_out:
+        copier = csv.DictWriter(f_out, fieldnames = reader.fieldnames)
+        copier.writeheader()
+        copier.writerows(data)
+
+
+def reader2data(reader):
```

Comment:
Did you write this? This could just as easily be replaced by the Python ``list`` builtin.

```python
james@basalt:~/code/git/crmprtd/crmprtd$ python3.5
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def an_iterator():
...     yield 'foo'
...     yield 'bar'
...     yield 'blah'
...     yield 'stuff'
... 
>>> for i in an_iterator():
...     print(i)
... 
foo
bar
blah
stuff
>>> print(list(an_iterator()))
['foo', 'bar', 'blah', 'stuff']
```